docs/source/using_doctr/running_on_aws.rst

```diff
@@ -0,0 +1,7 @@
+AWS Lambda
```

Comment:
we need to reference this in the `index.rst` otherwise it won't show up in the documentation